wildfly@.service

```diff
@@ -19,6 +19,8 @@ ExecStartPre=-/usr/bin/docker rm ${CONTAINER}
 ExecStartPre=/usr/bin/docker pull ${DOCKER_REPO}:${VERSION}
 
 ExecStart=/bin/bash -c 'docker run -P --name ${CONTAINER} --restart=always \
+  --hostname=${COREOS_PRIVATE_IPV4} \
```

Comment:
Let's change this to `--hostname %H` (%H will be replaced w/ the system's hostname by systemd). Someday that will become a more meaningful and unchanging value.